sdk/inc/azure/iot/az_iot_provisioning_client.h

```diff
@@ -423,6 +423,45 @@ AZ_NODISCARD az_result az_iot_provisioning_client_query_status_get_publish_topic
     size_t mqtt_topic_size,
     size_t* out_mqtt_topic_length);
 
+/**
+ * @brief Azure IoT Provisioning Client options for az_iot_provisioning_client_get_request_payload.
+ *        Not currently used.  Reserved for future use.
+ *
+ */
+typedef void az_iot_provisioning_client_payload_options;
```

Comment:
We typically create an "empty" struct for such options that we want to reserve for later. For example:

https://github.com/Azure/azure-sdk-for-c/blob/442b28d84d0349b5fec96c60a971d88f49500795/sdk/inc/azure/core/az_json.h#L265-L275